libs/shared/components/src/auth/UserMenu/UserButton.tsx

```diff
@@ -10,19 +10,17 @@ import { User } from '@island.is/shared/types'
 import { useLocale } from '@island.is/localization'
 import { userMessages } from '@island.is/shared/translations'
 import * as styles from './UserMenu.css'
+import { checkDelegation } from '@island.is/service-portal/core'
```

Comment:
`shared/components` should not depend on `service-portal/*`, can we move this utility somewhere else?

Reply:
Woops! I'll fix and move it to shared utils instead.